src/faqs/how-to-start-dsa.md

```diff
@@ -101,6 +104,22 @@ A big factor in interviews for DSA questions as well as in OA Rounds is **time**
 - [Atcoder Contests](https://atcoder.jp/contests/) - To start off, just give the 'Beginner Contests'.
 - [Codechef Contests](https://www.codechef.com/contests?itm_medium=navmenu&itm_campaign=allcontests) - Codechef has fallen off the throne for a while because of its degrading quality and user base, but it's still a decent site for beginners. To start off, I wouldn't recommend the Long contests because it has unnecessary bruteforce questions, instead just do the timed contests like Cook-off, Lunchtime, and Starters.
 
+#### Practice Platforms 
+
+|Name|Link|
+|:-|:-|
+|LeetCode|[leetcode.com](https://leetcode.com/)|
+|HackerRank|[hackerrank.com](https://www.hackerrank.com/)|
+|CodeSignal|[codesignal.com](https://codesignal.com/)|
+|GeeksforGeeks|[geeksforgeeks.org](https://www.geeksforgeeks.org/)|
+|Codeforces|[codeforces.com](https://codeforces.com/)|
+|TopCoder|[topcoder.com](https://www.topcoder.com/)|
+|InterviewBit|[interviewbit.com](https://www.interviewbit.com/)|
+|Codewars|[codewars.com](https://www.codewars.com/)|
+|Project Euler|[projecteuler.net](https://projecteuler.net/)|
+|Spoj|[spoj.com](https://www.spoj.com/)|
```

Comment:
Getting rid of HackerRank, Leetcode since they are already mentioned


```suggestion
|Name|Link|
|:-|:-|
|HackerRank|[hackerrank.com](https://www.hackerrank.com/)|
|CodeSignal|[codesignal.com](https://codesignal.com/)|
|TopCoder|[topcoder.com](https://www.topcoder.com/)|
|InterviewBit|[interviewbit.com](https://www.interviewbit.com/)|
|Codewars|[codewars.com](https://www.codewars.com/)|
|Project Euler|[projecteuler.net](https://projecteuler.net/)|
|Spoj|[spoj.com](https://www.spoj.com/)|
|GeeksforGeeks|[geeksforgeeks.org](https://www.geeksforgeeks.org/)|
```